packages/ckeditor5-dev-utils/lib/builds/getdllpluginwebpackconfig.js

```diff
@@ -32,6 +33,9 @@ module.exports = function getDllPluginWebpackConfig( webpack, options ) {
 	const packageName = tools.readPackageName( options.packagePath );
 	const langDirExists = fs.existsSync( path.join( options.packagePath, 'lang' ) );
 	const indexTsExists = fs.existsSync( path.join( options.packagePath, 'src', 'index.ts' ) );
+	const dependencies = Array.isArray( options.dependencies || [] ) ? options.dependencies || [] : [ options.dependencies ];
```

Comment:
This could be generated from `package.json` - listing all `@ckeditor/` dependencies.